backend/src/services/auth.js

```
@@ -107,9 +108,58 @@ export async function login(req) {
+         );
+      } else {
+         // User is in auth_internal (due to /owner target) but NOT an owner of any clients yet.
+         // Keep role as 'user' from DB. Context remains 'auth_internal'.
+         // poolMetadata should reflect they are a 'user' in this 'auth_internal' context for this request.
+         effectiveRole = user.role; // Should be 'user' from DB
+         req.session.role = user.role; // Keep DB role in session.role
+         // req.session.poolContext and req.session.schema are already 'auth_internal' from detectSchema
+         // Update poolMetadata to reflect true status for this login context
+         req.session.poolMetadata = {
+            user_role: user.role, // This will be 'user'
+            reason: "login_auth_internal_user_not_yet_owner",
```

Biome (1.9.4)

[error] This property value named user_role is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named user_role


(lint/suspicious/noDuplicateObjectKeys)